requirements.txt

```diff
@@ -16,6 +16,7 @@ jupyter
 openpyxl
 qtconsole<5.4.2
 napari
+acvl_utils<0.2.1
```

Comment:
I chose to do `acvl_utils!=0.2.1`, since I'm assuming that if `acvl_utils==0.2.2` will be released, it will include a bugfix? Maybe this is overly confident, but it would save us the hassle of having to update this once more in the future.

(It might also be good to add a comment explaining the limitation? SCT's requirements.txt is peppered with comments, though I guess `git blame` functions similarly to find out the reasons for each limitation.)

Reply:
Ah, funnily enough, the fix is already out: https://github.com/MIC-DKFZ/acvl_utils/issues/2#issuecomment-2470346349

Reply:
Ah great! In that case, your solution is likely best